### DISCUSSION TITLE
Any Select Component Problem?

### DISCUSSION THREAD

**Discussion by bogordesaincom** [mantinedev/mantine#440] [Q&A] [answered]
I have problem with Select Component when load old value, first time is not show, in second time that  show, any problem with my code?

https://user-images.githubusercontent.com/51335197/142745371-21b70727-59c6-4eaf-8416-88497eddf15e.mp4

```js
useImperativeHandle(ref, () => ({

        LoadFirstData(value) {

            form.setValues({ 
                id: value.id,
                first_name: value.first_name, 
                last_name: value.last_name, 
                phone: value.phone,
                email: value.email, 
            })
            setRoleValue(value.roles_name)

            setStatusValue(value.status.toString())
.....

        },

    }))

```

**Comment by rtivital** [comment, marked as answer]
Can you explain more? I can see that select has "Active" value in both cases

**Reply by bogordesaincom** [reply]
Not a Status field but for Role Field, on first that not show, but on second time this show / selected

**Reply by rtivital** [reply]
Cannot help much based on your code and demo, here is an example with async multi select values setting – https://codesandbox.io/s/white-thunder-ynxl5?file=/src/App.tsx

Make sure that MultiSelect has a correct `data` prop that includes everything that you specify in `value`

**Reply by bogordesaincom** [reply]
i was get solution forget about useeffect hahah
